server/routes/deleteFave.js

```diff
@@ -0,0 +1,20 @@
+const router = require('express').Router();
+const db = require('../../db/db');
+
+router.post('/', (req, res) => {
+    const topicId = Number(req.body.topic_id);
+    const userId = Number(req.body.user_id);
+    db.deleteFavorite(topicId,userId,(err, data) => {
+        db.getFavoritedTopics(userId, (err, data) => {
+            if (err) {
+                res.status(404).end()
```

Comment:
Good call on closing the connection if the user doesn't have the topic favorited.